File: backend/src/main/java/com/ssafy/backend/domain/auth/dto/LoginResponseDto.java
package com.ssafy.backend.domain.auth.dto;

import io.swagger.v3.oas.annotations.media.Schema;

public record LoginResponseDto(
    @Schema(description = "액세스 토큰", example = "<JWT>") String accessToken,
    @Schema(description = "토큰 타입", example = "Bearer") String tokenType,
    @Schema(description = "만료 시간(초)", example = "900") long expiresIn) {}

File: mobile/app/src/main/java/com/ssafy/mobile/feature/login/data/dto/LoginResponseDto.kt
package com.ssafy.mobile.feature.login.data.dto

import com.google.gson.annotations.SerializedName

data class LoginResponseDto(
    @SerializedName("accessToken")
    val accessToken: String,
    @SerializedName("tokenType")
    val tokenType: String,
    @SerializedName("expiresIn")
    val expiresIn: Long,
)
